consoleme/lib/password.py

```diff
@@ -0,0 +1,54 @@
+import asyncio
+from typing import Dict, List, Optional, Union
+
+from password_strength import PasswordPolicy
+
+from consoleme.config import config
+from consoleme.lib.redis import RedisHandler
+
+red = RedisHandler().redis_sync()
+
+
+async def wait_after_authentication_failure(user) -> str:
```

Comment:
We will need to revisit this later. I think we may still have issues if there are significant login attempt as this function sleeps for some time and pile up functions to reschedule in the event loop.